### DISCUSSION TITLE
How to scale inference service manually?

### DISCUSSION THREAD

**Discussion by yuzisun** [kubeflow/kfserving#1993] [Q&A]
Hey, I'm trying to reverse engineering to the kpa to understand what are the api calls the kpa performs in order to scale up or down an inference service so I can do it manually. This inference service has min 1 replicas and 100 max. Let's say I want to scale it manually to 2 inference services pods.. I can't seem to figure out how to do it. Scaling the deployment the revision created did not work.. Any suggestions??

originally posted by @daganida88  https://github.com/kserve/kserve/issues/1990

**Comment by daganida88** [comment]
Please assist :)

**Comment by daganida88** [comment]


**Reply by daganida88** [reply]
I added the following annotation inside the revision object:     autoscaling.knative.dev/target: "2". my bad.
